src/main.rs

```diff
@@ -145,7 +145,8 @@ fn check_for_failed_disks(
                 let mut dev_path = PathBuf::from("/dev");
                 let dev_name = state_machine.block_device.device.name.clone();
 
-                dev_path.push(state_machine.block_device.device.name);
+               // dev_path.push(state_machine.block_device.device.name);
```

Comment:
So I would just remove this commented line and if you want you can change the line above to remove the .clone() like this:
```
                let dev_name = &state_machine.block_device.device.name;
                dev_path.push(&dev_name)
```